bin/MadGraph5_aMCatNLO/cards/Zp_s2s2/Zp_s2s2_run_card.dat

```diff
@@ -44,8 +44,8 @@
 #*********************************************************************
 # PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
 #*********************************************************************
-     nn23lo1    = pdlabel     ! PDF set                                     
-     #230000    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
+     #nn23lo1    = pdlabel     ! PDF set                                     
+     230000    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```

Comment:
Oh I think I could have messed up here. Should I actually change it to `nn23lo1 = pdlabel`?

Reply:
lhapdf = pdlabel
$DEFAULT_PDF_SETS = lhaid

These two lines are correct settings

Reply:
Thanks @sihyunjeon!